ground/gcs/src/plugins/uavobjectutil/uavobjectutilmanager.cpp

```diff
@@ -705,8 +705,8 @@ bool UAVObjectUtilManager::descriptionToStructure(QByteArray desc, deviceDescrip
        struc.uavoHash.clear();
        struc.uavoHash=desc.mid(60,20);
 
-       // The git tag format used for releases is yyyymmdd
-       QRegularExpression certifiedPattern("^20[0-9]{6}$");
+       // The git tag format used for releases is Release-yyyymmdd
+       QRegularExpression certifiedPattern("^Release-20[0-9]{6}$");

```

Comment:
can we add `(\.[0-9])?`

That way I can fail to do a release/tag properly and try again on the same day :)